src/theme/palette.ts

```diff
@@ -30,17 +44,34 @@ const lightPalette: PaletteOptions | undefined = {
     main: "#06D7D7",
     dark: "#037777",
     light: "#83EBEB",
+    states: {
+      outlinedRestingBorder: "#06D7D7",
```

Comment:
In a future PR, can we go back to using human readable color names, especially since we have a lot of reuse here? eg `COLORS.SEAFOAM500` or whatever

Reply:
Will do!